src/main/java/uk/gov/hmcts/darts/task/runner/impl/CaseExpiryDeletionAutomatedTask.java

```diff
@@ -54,15 +50,10 @@ public AutomatedTaskName getAutomatedTaskName() {
     @Override
     @Transactional
     public void runTask() {
-        final List<CourtCaseEntity> courtCaseEntities = new ArrayList<>();
-
         caseRepository.findCasesToBeAnonymized(currentTimeHelper.currentOffsetDateTime(), Limit.of(getAutomatedTaskBatchSize()))
```

Comment:
What does this task do now?

Reply:
This find's cases to be anonymised and anonymises them

Reply:
So what saves the entity?

Reply:
oh ok didnt see the database save in the anonymiser class